cmd/livepeer/livepeer.go

```diff
@@ -219,7 +252,7 @@ func main() {
 		n.EthPassword = *ethPassword
 
 		if *deposit > 0 {
```

Comment:
What's the point of the deposit flag? Just a timesaver? I feel like we could keep starting the node separate from depositing, and instead just use the cli to deposit.

Reply:
So the deposit flag was added before the cli wizard was created to allow a broadcaster to optionally deposit tokens, but now that the cli wizard can deposit tokens, we shouldn't need this flag anymore

Reply:
Created https://github.com/livepeer/go-livepeer/issues/95 to address this